docs/Makefile

```diff
@@ -28,10 +25,14 @@ lint: $(DOCBOX_ID_FILE)
 	$(DOCKERMAKE) container-lint
 
 # run the docs in development mode
+#
+# requires DOCKERPORTS, which should not be shared with other targets due to
+# port collision. E.g. If other targets use DOCKERPORTS, multiple `make docs`
+# cannot run at the same time on the same server, which CI doesn't like --2020-03 walt
 .PHONY:run
 run: $(DOCBOX_ID_FILE) $(RUN_CFG)
 	@echo "\n\nOpen http://localhost:$(PORT)/ in your local browser\n\n"
-	$(DOCKERMAKE) container-run RUN_CFG=$(RUN_CFG)
+	docker run $(DOCKERFLAGS) $(DOCKERPORTS) $(DOCBOX) make BUILDROOT=$(BUILDROOT) container-run RUN_CFG=$(RUN_CFG)
```

Comment:
I'm a bit confused how this is solving the collision issue with neither PORT nor DOCKERPORTS accepting overrides from outside - can you clarify?

Reply:
The problem was that previously DOCKERMAKE included DOCKERPORTS... when it didn't need to.  This means both `make docs` and `make docs-lint` would unnecessarily put a claim on port 6601.

We don't use `make run-docs` in any CI automation, so it doesn't matter if this target hogs the port.